src/scrcmd.c

```diff
@@ -2302,3 +2302,23 @@ bool8 ScrCmd_warpsootopolislegend(struct ScriptContext *ctx)
     ResetInitialPlayerAvatarState();
     return TRUE;
 }
+
+bool8 ScrCmd_setdoublewildbattle(struct ScriptContext *ctx)
+{
+    u16 species1 = ScriptReadHalfword(ctx);
+    u8 level1 = ScriptReadByte(ctx);
+    u16 item1 = ScriptReadHalfword(ctx);
+    u16 species2 = ScriptReadHalfword(ctx);
+    u8 level2 = ScriptReadByte(ctx);
+    u16 item2 = ScriptReadHalfword(ctx);
+
+    CreateScriptedDoubleWildMon(species1, level1, item1, species2, level2, item2);
+    return FALSE;
+}
+
+bool8 ScrCmd_dodoublewildbattle(struct ScriptContext *ctx)
```

Comment:
I think this should have some protection, eg:
```
bool8 ScrCmd_dodoublewildbattle(struct ScriptContext *ctx)
{
    if (CountPartyAliveNonEggMons() > 1)
    {
        BattleSetup_StartScriptedDoubleWildBattle();
        ScriptContext1_Stop();
        return TRUE;
    }
}
```
or default to a single wild battle instead

Reply:
While normally I'd agree with something like that, its not the way double battles with a single party member is handled elsewhere.

Using the gym battle for Tate & Liza as an example, there's a check for how many party members the player has before triggering the battle in the script, not as part of the script command.

In any case, I'll go make an issue for how to handle the double battles with a single party member edgecase, since later generations let those battles start as intended.

Reply:
My above suggestion is relevant to this topic.